packages/private-build-infra/addon/index.ts

```diff
@@ -1,7 +1,3 @@
-/**
-  @module @ember-data/private-build-infra
-*/
-
 import { has } from 'require';
```

Comment:
Why are we removing this?

Reply:
we're removing it cause we don't document this package directly in the api docs so having yuidocs comments for it just means we accidentally create an empty package we wouldn't otherwise have